README.md

```diff
@@ -13,7 +15,53 @@ Used together, you get a powerful tool for developing high-performance angular a
 
 This repository holds a set of helpers to create **fully reactive** as well as **fully zone-less** applications.
 
-[![rx-angular logo](https://raw.githubusercontent.com/rx-angular/rx-angular/main/docs/images/rx-angular_logo.png)](https://www.rx-angular.io/)
+## Benefits
+
+ - 🔥 It's fast & performance focused: exceptional runtime speed & small bundle size
+ - ✔ Easy upgrade paths: migration scripts included since beta! `ng update @rx-angular/{cdk | template | state}`
+ - ✔ Lean and simple: No boilerplate guaranteed
+ - ✔ Well typed and tested
+ - ✔ Backwards compatible: support for Angular > v11
+
+> **❗ Warning**
+> Expect no migration scripts for any change in `experimental` folders
+## Used by
+<table style="width:100%">
+  <tr>
+    <td><img src="https://clickup.com/landing/images/logo-clickup_color.svg"></td>
+    <td><img src="https://media-exp1.licdn.com/dms/image/C4D0BAQEexCZaCyeDYg/company-logo_200_200/0/1626433899897?e=1652918400&v=beta&t=zjTJehE51V39-IaRRaaK_uBzLqcdTr2wsm7YpLLleUQ"></td>
+    <td><img src="https://avatars.githubusercontent.com/u/1733746?s=200&v=4"></td>
+  </tr>
+   <tr>
+    <th>Large scale application</th>
+    <th>Medium size project</th>
+    <th>Small project</th>
+  </tr>
+  <tr>
+    <td>
+      Url: clickup.com <br/>
+      LOC: <br/>
+      Used in Platforms: Web<br/>
+      Total Components: <br/>
+      Total Services: <br/>
```

Comment:
should we add all this stuff to the table if there is no data for any of them? ^^